core/src/test/scala/org/apache/spark/rpc/netty/NettyRpcEnvSuite.scala

```diff
@@ -42,6 +50,34 @@ class NettyRpcEnvSuite extends RpcEnvSuite with MockitoSugar with TimeLimits {
     new NettyRpcEnvFactory().create(config)
   }
 
+  test("SPARK-31233: Send message to clientMode RpcEnv with timeout") {
```

Comment:
maybe i'm missing something, but why not create a RpcEnv with clientMode = true?

Reply:
Hi @jiangxb1987 got it. The unit test is updated.